src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -140,6 +148,11 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin, EnumMixin)
     return this.isTypePutBackOn({ name: this.name })
   }
 
+  /** Return true if this filing is an administrative dissolution or put back on */
+  private get isAdministrativeDissolutionOrPutBackOn (): boolean {
+    return (this.isTypeAdministrativeDissolution({ name: this.name }) || this.isTypePutBackOn({ name: this.name }))
```

Comment:
You already have 2 getters for these conditions (and getter values are cached), so it would be better to just use your getters here 😀

Reply:
reverted!